libs/designsystem/form-field/src/input/input.component.scss

```diff
@@ -34,6 +34,10 @@ $padding-block-size-md: shared.$form-field-input-padding * 0.5;
   padding-block: $padding-block-size-md;
 }
 
+:host(.md.error) {
+  padding-block: calc(#{$padding-block-size-md} - 1px);
```

Comment:
Should we cherry-pick this change and create a separate PR for this?
This fixes the bug with a similar layout-shift happening when error state is toggled, but is not related to the message feature.